.env.sample.b2c

```diff
@@ -1,11 +1,9 @@
 # Note: If you are using Azure App Service, go to your app's Configuration,
 # and then set the following values into your app's "Application settings".
-
-FLASK_DEBUG=True
-# Expects the display name such as "contoso"
-TENANT_NAME=<tenant name>
 CLIENT_ID=<client id>
 CLIENT_SECRET=<client secret>
+# Expects the display name such as "contoso"
+TENANT_NAME=<tenant name>
```

Comment:
Ditto

Reply:
Just so you know, the order change in b2c is unnecessary because the tenant name is still a required parameter. That being said, I still enjoy the visual similarity between the two files. Let's keep your change here.